airgun/entities/contentview.py

```diff
@@ -239,12 +239,6 @@ def remove_version(self, entity_name, version_name, completely=True,
         view.confirm_remove.click()
         view.flash.assert_no_error()
         view.flash.dismiss()
-        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
-        result = view.versions.search(version_name)
-        if completely and not result:
-            return
```

Comment:
Use the same idiom you use for `Table`s in Robottelo:
```python
if completely and result[0]['Version'] != version_name:
    return
```

Reply:
This was working until I was getting the naviagation issue. It was not navigating to next view.

Reply:
On which test? I tried `test_positive_remove_cv_version_from_default_env` and `test_positive_remove_promoted_cv_version_from_default_env` - both passed for me locally, on latest 6.6 snap.

Reply:
let me try again.